pkgs/by-name/pd/pdns/package.nix

```diff
@@ -61,26 +66,29 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.enableFeature true "reproducible")
     (lib.enableFeature true "tools")
     (lib.enableFeature true "ixfrdist")
-    (lib.enableFeature true "systemd")
+    (lib.enableFeature stdenv.hostPlatform.isLinux "systemd")
```

Comment:
```suggestion
    (lib.enableFeature (lib.meta.availableOn stdenv.hostPlatform systemd) "systemd")
```

Reply:
The same applies for tinycdb

Reply:
Thank you for your suggestions, I added them.